### PR TITLE
Nightmare is over

### PR DESCRIPTION
**Pull request by hantariancuta**

### CODE REVIEW

File: src/index.pug [hunk 1]
```diff
@@ -1,5 +1,10 @@
 doctype html
 html
   head
+    title Studiu in web development
+    meta(
+      name="description"
+      content="Azi invatam programare in echipa prin zoom si exemple. "
```
Comment (marianzburlea) [inline src/index.pug]: Remove the extra space at the end

File: src/index.pug [hunk 2]
```diff
@@ -6,5 +6,33 @@ html
...
-    h1 Bine ati venit pe noul meu website
+    header
+      a(href="/") Acasa
+      a(href="/studiu-html") Studiu HTML
+      a(href="/studiu-css") Studiu CSS
+      a(href="/studiu-javascript") Studiu JavaScript
+
+    nav  
+      h3 HTML si CSS
+      a(href="/html") Studiu HTML
```
Comment (marianzburlea) [inline src/index.pug]: The links are like in the movie. Your project has /studiu-html as link, not /html :)

File: src/index.pug [hunk 3]
```diff
@@ -6,5 +6,33 @@ html
...
+      a(href="/css") Studiu CSS
```
Comment (marianzburlea) [inline src/index.pug]: /css -> /studiu-css

File: src/index.pug [hunk 4]
```diff
@@ -6,5 +6,33 @@ html
...
+      a(href="/javascript") Studiu JavaScript
+
+    article
+      h1 Pagina web HTML
+      p  Cu HTML poti sa creezi pagina ta web.Limbajul HTML consta intr-o serie de etichete numite si tag-uri care permit formatarea modului de afisare a informatiei din pagina Web(folosirea unor anumite fonturi, culori si stiluri pentru text, inserarea unor imagini sau a unor tabele, utilizarea listelor si a formularelor, etc). #[a(href="/html") Verifica!]
```
Comment (marianzburlea) [inline src/index.pug]: /html -> /studiu-html

Comment (marianzburlea) [reply src/index.pug]: web.Limbajul -> web. Limbajul
you always add a space after you end a sentence with a . and there is more content after that.

File: src/index.pug [hunk 5]
```diff
@@ -6,5 +6,33 @@ html
...
+      h2 Pagina web CSS
+      p  Cu CSS poti stiliza pagina web.Este un limbaj care defineste "layout-ul" pentru documentele HTML.CSS acopera culori, font-uri, margini, linii, inaltime, latime, imagini de fundal, pozitii avansate si multe alte optiuni.
+      h2 Pagina web JavaScript
+      p  Cu JavaScript poti sa creezi efecte interactive cu browserul. JavaScript este un limbaj de programare care iti permite sa implementezi lucruri complexe pe paginile web - de fiecare data cand o pagina web face mai mult decat sa afiseze informatii statice pentru ca tu sa te uiti - afisand actualizari de continut in timp util sau harti interactive, grafice animate 2D/3D, video jukebox, etc.JavaScript permite crearea unor aplicatii care sa ruleze in internet. #[a(href="/css") Click aici!]
+      h2 Pagina web
+      p  Prin exemple si exercitii se studiaza eficient limbajele de programare. #[a(href="/javascript") Descopera!]
```
Comment (marianzburlea) [inline src/index.pug:26]: /javascript -> /studiu-javascript

File: src/style.scss [hunk 6]
```diff
@@ -5,4 +5,19 @@
   --color-black: #000;
   --color-seashel: #f1f1f1;
   --color-silver: #ccc;
+}
+
+html, body {
+  height: 100%;
+
+  * {
```
Comment (marianzburlea) [inline src/style.scss]: * means select everything
it has no place in the html, body selector, it should be outside of it.

File: src/style.scss [hunk 7]
```diff
@@ -1,5 +1,2 @@
-// @import '_component/config/color';
-// @import '_component/config/typography';
-// @import '_component/layout/layout';
-// @import '_component/header/header';
-// @import '_component/footer/footer';
\ No newline at end of file
+h1 {color: blue;
```
Comment (marianzburlea) [inline src/style.scss]: Although it's OK, I want you to move the color: blue on a new line to increase readability

File: src/index.pug [hunk 8]
```diff
@@ -21,13 +21,13 @@ html
 
     article
       h1 Pagina web HTML
-      p  Cu HTML poti sa creezi pagina ta web.Limbajul HTML consta intr-o serie de etichete numite si tag-uri care permit formatarea modului de afisare a informatiei din pagina Web(folosirea unor anumite fonturi, culori si stiluri pentru text, inserarea unor imagini sau a unor tabele, utilizarea listelor si a formularelor, etc)
```
Comment (marianzburlea) [inline src/index.pug]: Add space after dots !!

File: src/index.pug [hunk 9]
```diff
@@ -28,6 +28,8 @@ html
       p  Cu JavaScript poti sa creezi efecte interactive cu browserul. JavaScript este un limbaj de programare care iti permite sa implementezi lucruri complexe pe paginile web - de fiecare data cand o pagina web face mai mult decat sa afiseze informatii statice pentru ca tu sa te uiti - afisand actualizari de continut in timp util sau harti interactive, grafice animate 2D/3D, video jukebox, etc.JavaScript permite crearea unor aplicatii care sa ruleze in internet.
       h2 Pagina web
       p  Prin exemple si exercitii se studiaza eficient limbajele de programare 
+
+      footer &copy; 2019 ANCUTA HANTARI Web development sudent
```
Comment (marianzburlea) [inline src/index.pug]: student

File: src/index.pug [hunk 10]
```diff
@@ -20,13 +20,13 @@ html
...
     article
-      h1 Pagina web
-      p  Cu HTML poti sa creezi pagina ta web
-      h1 Pagina web
-      p  Cu CSS poti stiliza pagina web
-      h1 Pagina web
-      p  Cu JavaScript poti sa creezi efecte interactive cu browserul
-      h1 Pagina web
+      h1 Pagina web HTML
+      p  Cu HTML poti sa creezi pagina ta web.Limbajul HTML consta intr-o serie de etichete numite si tag-uri care permit formatarea modului de afisare a informatiei din pagina Web(folosirea unor anumite fonturi, culori si stiluri pentru text, inserarea unor imagini sau a unor tabele, utilizarea listelor si a formularelor, etc)
```
Comment (marianzburlea) [inline src/index.pug]: space after dot

File: src/index.pug [hunk 11]
```diff
@@ -17,4 +17,19 @@ html
       h3 HTML si CSS
       a(href="/html") Studiu HTML
```
Comment (marianzburlea) [inline src/index.pug]: /html -> /studiu-html

File: src/index.pug [hunk 12]
```diff
@@ -17,4 +17,19 @@ html
...
       a(href="/css") Studiu CSS
-      a(href="/javascript") Studiu JavaScript
\ No newline at end of file
+      a(href="/javascript") Studiu JavaScript
```
Comment (marianzburlea) [inline src/index.pug]: /javascript -> /studiu-javascript

File: src/index.pug [hunk 13]
```diff
@@ -7,7 +7,14 @@ html
...
\ No newline at end of file
+    header
+      a(href="/") Acasa
+      a(href="/studiu-html") Studiu HTML
+      a(href="/studiu-css") Studiu CSS
+      a(href="/studiu-javascript") Studiu JavaScript
+
+    nav  
+      h3 HTML si CSS
+      a(href="/html") Studiu HTML
```
Comment (marianzburlea) [inline src/index.pug]: /html -> /studiu-html

**Review by marianzburlea — CHANGES_REQUESTED** [review]

File: src/index.pug (hunk 2 above)
Comment (hantariancuta) [reply src/index.pug]: ;-)

File: src/index.pug (hunk 4 above)
Comment (hantariancuta) [reply src/index.pug]: Waww

File: src/style.scss (hunk 6 above)
Comment (hantariancuta) [reply src/style.scss]: aaa, now I understand

Comment (hantariancuta) [reply src/style.scss]: So, as I did have no idea how I can do it, I looked up on google and I've found exactly on w3schools it does say about addind a div element for height. Here is the link where I've found it.
image

File: src/style.scss (hunk 7 above)
Comment (hantariancuta) [reply src/style.scss]: I do not have this anymore in my code. I deleted it.

File: src/index.pug (hunk 10 above)
Comment (hantariancuta) [reply src/index.pug]: I've did this, already!

**Pull request merged**